src/helpers/helpers.core.js

```diff
@@ -26,28 +33,27 @@ export function clampAll(obj, from, to) {
  * @param {Point} point
  * @param {Point} center
  * @param {number} radius
- * @param {number} borderWidth
+ * @param {number} hitSize
  * @returns {boolean}
  */
-export function inPointRange(point, center, radius, borderWidth) {
+export function inPointRange(point, center, radius, hitSize) {
   if (!point || !center || radius <= 0) {
     return false;
   }
-  const hBorderWidth = borderWidth / 2;
-  return (Math.pow(point.x - center.x, 2) + Math.pow(point.y - center.y, 2)) <= Math.pow(radius + hBorderWidth, 2);
+  return (Math.pow(point.x - center.x, 2) + Math.pow(point.y - center.y, 2)) <= Math.pow(radius + hitSize, 2);
 }
 
 /**
  * @param {Point} point
  * @param {{x: number, y: number, x2: number, y2: number}} rect
  * @param {InteractionAxis} axis
- * @param {number} borderWidth
+ * @param {{borderWidth: number, hitTolerance: number}} hitsize
  * @returns {boolean}
  */
-export function inBoxRange(point, {x, y, x2, y2}, axis, borderWidth) {
-  const hBorderWidth = borderWidth / 2;
-  const inRangeX = point.x >= x - hBorderWidth - EPSILON && point.x <= x2 + hBorderWidth + EPSILON;
-  const inRangeY = point.y >= y - hBorderWidth - EPSILON && point.y <= y2 + hBorderWidth + EPSILON;
+export function inBoxRange(point, {x, y, x2, y2}, axis, {borderWidth, hitTolerance}) {
+  const hitSize = borderWidth / 2 + hitTolerance / 2;
```

Comment:
Personally I find this a bit clearer/more readable and I se that you also do it like this later on

```suggestion
  const hitSize = (borderWidth + hitTolerance) / 2;
```

Reply:
Thanks @LeeLenaleee . I'm going to apply your reviews (on other PRs) asap. Unfortunately these PRs are quite old and I don't remember well all code I wrote. :(